anago

```diff
@@ -742,6 +743,23 @@ get_build_candidate () {
                                 $PARENT_BRANCH \
    || return 1
 
+  # When creating alphas, make sure that X.X.0 has been created first or warn.
```

Comment:
Can we just ask the user if (s)he wants to do fast-forward in the future and if so disallow the creation of the alpha tag?